### PR TITLE
Explain Issues with Multi-Process Servers

### PR DESCRIPTION
**Pull request by seanpdoyle**
Move the details of [#94] to the README.

[#94]: https://github.com/thoughtbot/ember-cli-rails/issues/94

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,6 +234,18 @@ example, `/ember-tests`) and the name of the Ember app.
...
+## Serving from multi-process servers in development
+
+If you're using a multi-process server ([Puma], [Unicorn], etc.) in development,
+make sure it's configured to run a single worker process.
+
+Without restricting the server to a single process, [it is possible for multiple
+EmberCLI runners to clobber each others' work][#94].
+
+[Puma]: https://github.com/puma/puma
+[Unicorn]: https://rubygems.org/gems/unicorn/versions/4.9.0
```
Comment (tute) [inline README.md]: Do we need to link to a specific version?

**Comment by tute** [comment]
One question, LGTM!

File: README.md (hunk 1 above)
Comment (seanpdoyle) [inline README.md]: Nope, this was the first URL I landed on when I googled the gem.

How do you feel about linking to https://rubygems.org/gems/unicorn instead?

Comment (tute) [inline README.md]: > How do you feel about linking to https://rubygems.org/gems/unicorn instead?

That's perfect.

**Pull request merged**